Задание для кандидатов. Выполнить ревью.sql

```diff
@@ -0,0 +1,123 @@
+create procedure syn.usp_ImportFileCustomerSeasonal
+	@ID_Record int
+AS
+set nocount on
+begin
+	declare @RowCount int = (select count(*) from syn.SA_CustomerSeasonal)
+	declare @ErrorMessage varchar(max)
+
+-- Проверка на корректность загрузки
+	if not exists (
+	select 1
+	from syn.ImportFile as f
+	where f.ID = @ID_Record

```

Comment:
Комментарий 4
перечисление идет внутри скобок функции
```suggestion
	     select 1
	     from syn.ImportFile as f
	     where f.ID = @ID_Record

```